CMakeLists.txt

```diff
@@ -177,3 +177,11 @@ TARGET_LINK_LIBRARIES(wasm-dis passes support asmjs)
 SET_PROPERTY(TARGET wasm-dis PROPERTY CXX_STANDARD 11)
 SET_PROPERTY(TARGET wasm-dis PROPERTY CXX_STANDARD_REQUIRED ON)
 INSTALL(TARGETS wasm-dis DESTINATION bin)
+
+# Deploy scripts that are utilized at runtime to bin/ directory, so that the build directory can be packaged as a standalone toolchain.
+FILE(COPY "${CMAKE_CURRENT_SOURCE_DIR}/scripts" DESTINATION "${PROJECT_BINARY_DIR}/bin/")
```

Comment:
I still think having install targets is the 'right' way to do this.
When creating a distribution, we don't want do depend on a source dir, and I don't think we want this stuff in the `bin` dir; we want it in some other dir: probably `lib`, `share` or some subdir of one of those. For local testing, I don't care too much; we can add something to `check.py` to handle things to avoid having to run the install step if we want, I'm happy to always do out-of-tree builds, or whatever.